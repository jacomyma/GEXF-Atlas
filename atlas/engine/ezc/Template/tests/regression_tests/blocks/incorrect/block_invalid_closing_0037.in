{* file: block_invalid_closing_0037.in *}
{var $foo = "T42p"}
{/switch}

