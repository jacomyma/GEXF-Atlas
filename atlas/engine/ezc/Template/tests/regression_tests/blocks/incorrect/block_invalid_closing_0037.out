mock:3:10: Found closing block {/switch} without an opening block.

{/switch}
         ^
